src/design/demo/fallback-feature-images.twig

```diff
@@ -1,5 +1,6 @@
-
-{% embed '@cloudfour/objects/deck/deck.twig' %}
+{% embed '@cloudfour/objects/deck/deck.twig' with {
+  class: 'o-deck--2-column@l'
```

Comment:
This `class` was previously specified via the story in `args`, but I thought that was confusing. Moving it here means there's only one file to edit for this demo.